From c9aaa169479b077eff1eb071a0f6fdc978084088 Mon Sep 17 00:00:00 2001
From: yachun <yachun@marvell.com>
Date: Wed, 13 Sep 2017 17:13:02 +0800
Subject: [PATCH 1239/1345] fix: dts: a3900: remove pcie comphy specification
 for avoid network error

commit  2533c9ea630f3b12334574bb141c6575ec515463 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

PCIe comphy driver issues: The PPv22 mac0 and mac2 would stop functioning,
                           when specify the PCIEx2 in the DTS.

This patch is a WA for this issue, which mvppv2 will stop functioning
with the PCIEx2 setting in DTS. Remove the pcie comphy specification
and depend on uboot PCIe initialization, the network and PCIE are works.

Please remove this patch after fix the PCIe comphy driver issues.

Change-Id: I7cb1606b251482f1bbf5124805c74fcb7c125b6d
Signed-off-by: yachun <yachun@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/44421
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/marvell/armada-3900-db-axis.dts |    4 ----
 arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts  |    8 --------
 arch/arm64/boot/dts/marvell/armada-3900-db.dtsi    |    2 --
 3 files changed, 14 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts
index 2d1b1f6..83d0013 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-axis.dts
@@ -66,10 +66,6 @@
 			};
 		};
 
-		pcie@0x600000 {
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
-		};
 	};
 
 	cpm_reg_usb3_vbus0: cpm_usb3_vbus@0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
index 05ce78e..e635f5a 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
@@ -61,14 +61,6 @@
 			};
 		};
 
-		pcie@0x600000 {
-			num-lanes = <2>;
-			phys = <&comphy0 0 COMPHY_PCIE0
-				&comphy0 1 COMPHY_PCIE0
-				>;
-			phy-names = "pcie-phy0", "pcie-phy1";
-
-		};
 	};
 
 	cpm_reg_usb3_vbus0: cpm_usb3_vbus@0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
index dbc62a4..4bc59f9 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
@@ -191,8 +191,6 @@
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 			ranges = <0x81000000 0 0xf7020000 0  0xf7020000 0 0x10000	/* downstream I/O */
 				  0x82000000 0 0xf8000000 0  0xf8000000 0 0xf00000>;	/* non-prefetchable memory */
 		};
-- 
1.7.9.5

